blog/content/post/2019-11-13-Websummit-2019.md

```diff
@@ -0,0 +1,81 @@
+to Lisbon, Web Summit is considered to be the largest tech event in the world.
+Until 2019 there was no representation of 3mdeb on Web Summit, but now we've
+decided, that it is the right time to send one.
+
+"Tech event" is the keyword that turns all of us on and makes us travel over
+thousand kilometers, but after the struggle I was the one who become the first
+official 3mdeb representative on that event. Full of hope and curiosity, with
+a great financial and organizational support provided by Export Broker, at
+Sunday, November 3, I've joined polish group on the Lech Walesa airport in
+Gdansk.
+
+Have you heard about the main types of a lie?
+
+In classical version there are small lies, big lies and statistics. I've found
+another - calling Web Summit a tech event. Blows up the scale. It was political,
+it was sociological, it was futuristic, but it has much more to present about
+the people relations, than about technology.
+
+The opening day was a series of presentations. The main guest wasn't physically
+with us, but his speech was streamed from Russia. Edward Snowden - the great
+warrior for freedom of speech, which had left western world and settled down
+in the capital of freedom - Moscow. Though I don't consider him the most
+reliable man living, the speech itself was fine. In the end he started to accuse
+companies by their name for trying to enslave world population, and when he
+got to Huawei it appeared that he runned out of time. The next speaker was
+Guo Ping, Huawei's Rotating Chairman - astonishing coincidence, isn't it?
+He tried to persuade an audience that all the thing anyone heard about his
+company is a lie, no matter the facts. Hopefully there were a lot of young
+and inexperienced people there, in the other case the audience might not be
+persuaded.
+
+After the opening there should be a Night Summit event. But it was raining, and
+almost no one came.   
+
+Day two started with a something-like-fairs, but many of the companies
+exhibiting there had nothing but themselves for sale. At first I stayed
+at Polish Agency for Enterprise Development stand. I want to sincerely thank
+all the people there for helping me to present our products and let me to leave
+our materials on the stand. After several hours of explaining children what
+firmware and embedded are I've decided to stop wasting my time, and move forward
+to hunt some commercial pray. I've visited surroundings stands looking for
+someone who would understand what are we doing, why, and be interested in
+cooperation. It was tough. Web Summit is large. I was told that there were
+over 70.000 attenders and 12.000 CEO's. Founding someone who runs a REAL tech
+company suitable for us was a lot of walking and talking. Finally I've succeeded
+but in a bit unexpected way.
+
+[![Cervi Robotics](https://cloud.3mdeb.com/index.php/s/sokG2axR4RX9DCc=preview)](https://cloud.3mdeb.com/index.php/s/sokG2axR4RX9DCc)
```

Comment:
@lwcislo preview part return error, but video is very good, great idea and please talk with Staszek to push that to social media - Youtube etc.